### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by konzert**
Added the conversational quiz library and the landing page. For now, we're using axios as our http client in the front-end, and i've added a button that shows that when you click on it (and the backend is live) it'll print out all the answers for a certain quiz in the console, as a demonstration.

### CODE REVIEW

File: client/decs.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare module 'react-conversation-form';
```
Comment (vdrelciuc) [inline client/decs.d.ts:1]: What's the point of this one-line file?

**Comment by vdrelciuc** [comment]
Deprecated
Quite a few client dependencies are deprecated. Consider upgrading them when you can

File: client/src/components/stylesheets/homepage.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    top: 50%;
+    left: 50%;
+    color: white;
+    font-family: 'Ubuntu', sans-serif;
+    margin-top: -50px;
+    margin-left: -50px;
+    width: auto;
+    height: auto;
+    background-color: #5444b2;
+} */
```
Comment (vdrelciuc) [inline client/src/components/stylesheets/homepage.css:36]: No dead code please :)

File: client/src/components/global/Header.tsx [hunk 3]
```diff
@@ -4,10 +4,10 @@ import { LanguageContainer as LanguageFragment } from './Language';
 const Header: React.FunctionComponent = () => {
     return (
         <div>
-            <h1 style={{ display: 'inline-block' }}>Header</h1>
+            {/* <h1 style={{ display: 'inline-block' }}>Header</h1>
             <div style={{ display: 'inline-block' }}>
                 <LanguageFragment />
-            </div>
+            </div> */}
```
Comment (vdrelciuc) [inline client/src/components/global/Header.tsx:10]: You should't commit dead/commented code

File: .idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Comment (vdrelciuc) [inline .idea/.gitignore:5]: You should remove all .idea folders from the repo (and add them to .gitignore). That applies for everything under .idea/ and everything under /server/.idea/

File: client/src/components/stylesheets/quiz.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
+/* cf-chat scrollable {
+    height: 100%;
+    width: 100%;
+} */
```
Comment (vdrelciuc) [inline client/src/components/stylesheets/quiz.css:4]: No dead code please :)

**Comment by vdrelciuc** [comment]
One more thing: you might want to write at least some very basic unit tests

File: client/decs.d.ts (hunk 1 above)
Comment (amulyaparmar) [reply client/decs.d.ts:1]: This declare was to fix the fact that react-conversational-form is a naturally a typescript language so we have to declare the module in a separate file so we can tell the compiler you don't have to check for typescript

**Pull request merged**